### PR TITLE
Store dags cleanup 1 10

### PR DESCRIPTION
**Pull request by anitakar**
---
Make sure to mark the boxes below before creating PR: [x]

- [x] Description above provides context of the change
- [x] Unit tests coverage for changes (not needed for documentation changes)
- [x] Target Github ISSUE in description if exists
- [x] Commits follow "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)"
- [x] Relevant documentation is updated including usage instructions.
- [x] I will engage committers as explained in [Contribution Workflow Example](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#contribution-workflow-example).

---
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/master/UPDATING.md).
Read the [Pull Request Guidelines](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#pull-request-guidelines) for more information.

### CODE REVIEW

File: airflow/www_rbac/views.py [hunk 1]
```diff
@@ -564,7 +568,8 @@ def dag_details(self, session=None):
         dag_id = request.args.get('dag_id')
         dag_orm = DagModel.get_dagmodel(dag_id, session=session)
         # FIXME: items needed for this view should move to the database
-        dag = dag_orm.get_dag(STORE_SERIALIZED_DAGS)
+        dag = dag_orm.get_dag(
+            conf.getboolean('core', 'store_serialized_dags', fallback=False))
```
Comment (ashb) [inline airflow/www_rbac/views.py:572]: No. This does not make sense.

Thsi change just makes it slower. STORE_SERIALIZED_DAGS is `conf.getboolean('core', 'store_serialized_dags', fallback=False))`, but without calling the slow conf.getboolean every time.

File: airflow/www/views.py [hunk 2]
```diff
@@ -94,7 +94,16 @@
 
 UTF8_READER = codecs.getreader('utf-8')
 
-dagbag = models.DagBag(settings.DAGS_FOLDER, store_serialized_dags=STORE_SERIALIZED_DAGS)
+try:
+    async_dagbag_loader = conf.getboolean('webserver', 'async_dagbag_loader')
```
Comment (ashb) [inline airflow/www/views.py:98]: This is not a "clean-up" -- it is a whole new feature.

And it was a feature that was discussed and rejected in the original serialized DAG proposal/PR. Please to not bring it back in without a very clear indication of what problem it solves over the current method against  1.10.10.

**Review by ashb — CHANGES_REQUESTED** [review]

**Comment by ashb** [comment]
I'm going to close this, as the main feature here is not needed with Dag serialization on, and the other changes are a performance regression.

We can re-open this is you disagree.

File: airflow/www/views.py (hunk 2 above)
Comment (anitakar) [reply airflow/www/views.py:98]: Sorry for that. I thought it got lost or was removed. It is not necessary with dag serialization and definitely such big functionalities should be first discussed via. AIP

File: airflow/www_rbac/views.py (hunk 1 above)
Comment (anitakar) [reply airflow/www_rbac/views.py:572]: Sure. Airflow does not support dynamic settings. And even if it did, it should not be one of them.

**Pull request closed without merging**